test/javascript/components/interchange.js

```diff
@@ -170,12 +170,10 @@ describe('Interchange', function() {
     it('calls reflow on viewport size change once', function(done) {
       $html = $(generateTemplate('image')).appendTo('body');
       plugin = new Foundation.Interchange($html, {});
-      Foundation.IHearYou();
-      let spy = sinon.spy(plugin, '_reflow');
-
       setTimeout(function() {
-        $(window).trigger('resize');
+        Foundation.IHearYou();
```

Comment:
Why should `Foundation.IHearYou` be called dynamically ?

Reply:
Because it was already there and is also done like this in the other unit tests, see https://github.com/zurb/foundation-sites/pull/11179/files#diff-e0276fdab014830c6cb0753691ddc7f1L173

Reply:
I meant _dynamically ?_

Reply:
Because this seems to solve the race condition with the other changes (see the other PR which has all changes).

Reply:
The tests successfully ran with these changes. ;-)

Reply:
hmm... Ok. But we don't know _why_ ?
`IHearYou` works on `[data-resize}` which is set at `Interchange` Initialisation. This should be done synchronously and does not require any laps time.

Reply:
I got the "hey, it just works". But I don't think that's enough. We'll get into others trouble and weird bugs soon if we just to like "meh, ok it just works" without trying to understand where the real issue is.

Reply:
Ah yeah. `debounce` 10 by default.